packages/modular-scripts/src/__tests__/lint.test.ts

```diff
@@ -97,3 +101,81 @@ describe('Modular lint', () => {
     });
   });
 });
+
+describe('lint command can successfully do selective tests based on selected packages', () => {
+  const fixturesFolder = path.join(
+    __dirname,
+    Array.from({ length: 4 }).reduce<string>(
+      (acc) => `${acc}..${path.sep}`,
+      '',
+    ),
+    '__fixtures__',
+    'selective-lint',
+  );
+
+  let randomOutputFolder: string;
+
+  beforeEach(() => {
```

Comment:
Consider using non-execa (i.e. not relying on `runModularPipeLogs` and instead calling `lint()` directly) techniques like in the selective typecheck PR? Should be a bit faster, and test coverage should get tracked properly.

I think it's viable for `lint`, just a case of mocking the modular root after creating the temp modular repo. The typecheck example of this approach [is here](https://github.com/jpmorganchase/modular/blob/18bb7a9c31854e47910fe2c1d98ab39d28dda6ad/packages/modular-scripts/src/__tests__/typecheck.test.ts)

Reply:
Created a separate spike ticket for this.